src/ESPeasyGPIO.ino

```diff
@@ -0,0 +1,396 @@
+/****************************************************************************/
+// Central functions for GPIO handling
+// **************************************************************************/
+#include "EspEasyGPIO.h"
+
+//********************************************************************************
+// Internal GPIO write
+//********************************************************************************
+void GPIO_Internal_Write(byte pin, byte value)
+{
+  pinMode(pin, OUTPUT);
+  digitalWrite(pin, value);
+}
+
+//********************************************************************************
+// Internal GPIO read
+//********************************************************************************
+bool GPIO_Internal_Read(byte pin)
+{
+  return digitalRead(pin)==HIGH;
```

Comment:
Does not check using `checkValidPortRange`

Reply:
Can you help me identifying the valid port ranges for each platform?
8266: 0-5 & 12-16
8285: 0-5 & 10-16
ESP32: ??

And I cannot find anywhere `#if defined ESP8285`. 
Is it used?

Reply:
See the file `src/Hardware.ino`

Reply:
ok